nyc_data/ppe/migrations/0006_auto_20200410_1635.py

```diff
@@ -0,0 +1,24 @@
+# Generated by Django 3.0.5 on 2020-04-10 16:35
+
+import django.contrib.postgres.fields.jsonb
+from django.db import migrations, models
+import uuid
+
+
+def truncate_tables(apps, schema_editor):
```

Comment:
Part of the most recent migration appears to delete all rows in the Inventory, Delivery, and Purchase tables. Assuming that's because we want to "start fresh" on these tables, now that we're at a point where we'll have uploads occurring consistently?

Reply:
Yeah, it's because in the new table we need a foreign key to the import data. I didn't want to make it nullable so I just dropped the existing data first